scripts/standalone_embed.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Licensed to the LF AI & Data foundation under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+start() {
+    cat << EOF > embedEtcd.yaml
+listen-client-urls: http://0.0.0.0:2379
+advertise-client-urls: http://0.0.0.0:2379
+EOF
+
+    sudo docker run -d \
+        --name milvus-standalone \
+        --security-opt seccomp:unconfined \
+        -e ETCD_USE_EMBED=true \
+        -e ETCD_DATA_DIR=/var/lib/milvus/etcd \
+        -e ETCD_CONFIG_PATH=/milvus/configs/embedEtcd.yaml \
+        -e COMMON_STORAGETYPE=local \
+        -v ./volumes/milvus:/var/lib/milvus \
+        -v ./embedEtcd.yaml:/milvus/configs/embedEtcd.yaml \
+        -p 19530:19530 \
+        -p 9091:9091 \
+        -p 2379:2379 \
+        --health-cmd="curl -f http://localhost:9091/healthz" \
+        --health-interval=30s \
+        --health-start-period=90s \
+        --health-timeout=20s \
+        --health-retries=3 \
+        milvusdb/milvus:latest \
```

Comment:
We should use specific image tag instead of `latest`.

Reply:
already changed